pkg/client/results/manual.go

```diff
@@ -58,6 +58,7 @@ func manualProcessFile(pluginDir, currentFile string) (Item, error) {
 
 	rootObj.Status = resultObj.Status
 	rootObj.Items = resultObj.Items
+	rootObj.Details = resultObj.Details
```

Comment:
Including this as this field wasn't being set on the root item. It's covered in the new test in `manual_test.go`.